src/View/View.php

```diff
@@ -1259,7 +1259,7 @@ protected function _paths($plugin = null, $cached = true)
             $themePaths,
             $pluginPaths,
             $templatePaths,
-            [dirname(__DIR__) . DS . 'Template' . DS]
+            [CAKE . 'Template' . DS]
```

Comment:
@markstory while cleaning ``DS`` usage, I figured out that it is stillpart of the [core library](https://github.com/cakephp/cakephp/blob/master/src/Core/functions.php#L17-L23) (I didn't remember it either)


Are you still interested in replacing it, I can send my PR if you will.